_monitoring-your-cluster/pa/index.md

```diff
@@ -8,57 +8,106 @@ redirect_from:
   - /monitoring-plugins/pa/index/
 ---
 
-# Performance analyzer
+# Performance Analyzer
 
-Performance analyzer is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics. 
+Performance Analyzer is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics. 
 
-The performance analyzer plugin is installed by default in OpenSearch version 2.0 and higher.
+The Performance Analyzer plugin is installed by default in OpenSearch version 2.0 and higher. If you want to use OpenSearch 2.0 or later with Performance Analyzer disabled, see [Disable Performance Analyzer](#disable-performance-analyzer).
 {: .note }
 
-## Performance analyzer installation and configuration
+## Prerequisites
 
-The following sections provide the steps for installing and configuring the performance analyzer plugin.
+Review the following prerequisites before installing Performance Analyzer.
```

Comment:
I thought it was installed by default? Maybe something like, "To install the plugin independently of OpenSearch 2.0 or later, review the following prerequisites before installing it manually."

Reply:
A more accurate description to prevent the issue our user had might be "Before using Performance Analyzer with OpenSearch, review the following prerequisites.